tests/unit/batch_sys_handlers/test_slurm.py

```diff
@@ -15,9 +15,12 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 import pytest
+import os
 
 from cylc.flow.batch_sys_handlers.slurm import BATCH_SYS_HANDLER
 
+home = os.path.expandvars('$HOME/')
```

Comment:
On master the full path was given in the SBATCH directives in the job file, so I've made the test, and my changes consistent with that!

Reply:
Dammit, the tests were miss-leading us!